internal/datastore/postgres/common/common.go

```diff
@@ -0,0 +1,74 @@
+package common
```

Comment:
could name this file pgx.go

Reply:
addressed in https://github.com/authzed/spicedb/pull/774/commits/6981da212673943af881e315906a72f325ac8922